dist/gaia-header.js

```diff
@@ -913,11 +918,40 @@ module.exports = component.register('gaia-header', {
   observerStop: function() {
     if (!this.observing) { return; }
     this.observer.disconnect();
+    window.removeEventListener('resize', this);
+    this._resizeHandled = false;
+
     this.observing = false;
     debug('observer stopped');
   },
 
   /**
+   * Handle DOM events
+   */
+  handleEvent: function(e) {
+    switch(e.type) {
+      case 'resize':
+        this.onResize();
+        break;
+    }
+  },
+
+  /**
+   * handle 'resize' events
+   */
+  onResize: function() {
+    if (this._resizeHandled) {
+      return;
+    }
+
+    this._resizeHandled = true;
+    window.requestAnimationFrame(() => {
```

Comment:
yeah, for throttling: I just did as suggested on MDN:https://developer.mozilla.org/en-US/docs/Web/Events/resize#Example

I thought that just calling runFontFitSoon was not enough, because we could receive several "resize" event per seconds, but they wouldn't be in the same microtask.

I can add a comment for sure !